base/math.jl

```diff
@@ -34,6 +34,7 @@ using Base: sign_mask, exponent_mask, exponent_one, exponent_bias,
 
 using Core.Intrinsics: sqrt_llvm, powi_llvm
 
+typealias IEEEFloat Union{Float16,Float32,Float64}
```

Comment:
@tkelman  It may be more appropriate to place this in `float.jl` ?
For example the functions following from: https://github.com/JuliaLang/julia/blob/master/base/float.jl#L747
to
https://github.com/JuliaLang/julia/blob/master/base/float.jl#L753
(there are a few other functions like this as well)
only really make sense for ieee floats.